src/renderer/containers/Onboard/ConfigureValidator/ConfigureValidatorContainer.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {useEffect, useState} from "react";
+import {InputForm} from "../../../components/Input/InputForm";
+import {OnBoardingRoutes, Routes} from "../../../constants/routes";
+import {useHistory} from "react-router-dom";
+import {ButtonPrimary} from "../../../components/Button/ButtonStandard";
+import {wordlists} from "bip39";
+import {randBetween} from "@chainsafe/lodestar-utils";
+import {useDispatch} from "react-redux";
+import {setName as setNameAction} from "../../../ducks/register/actions";
+
+export const ConfigureValidatorContainer: React.FC = () => {
+    const history = useHistory();
+    const [name, setName] = useState("");
+    const [defaultName, setDefaultName] = useState("");
+
+    useEffect(() => {
+        const capitalize = (string: string): string => string.charAt(0).toUpperCase() + string.slice(1);
+
+        const englishWordList = wordlists["english"];
+        setDefaultName(
+            "Validator " +
+                capitalize(englishWordList[randBetween(0, englishWordList.length / 2)]) +
+                " " +
+                capitalize(englishWordList[randBetween(englishWordList.length / 2, englishWordList.length - 1)]),
+        );
+    }, []);
+
+    const dispatch = useDispatch();
+
+    const onSubmit = (): void => {
+        dispatch(setNameAction(name ? name : defaultName));
```

Comment:
Hmm I would rather set defaultName onChange. Because currently, if user would delete everything and clicked save it would silently save default name?